pyproject.toml

```diff
@@ -182,7 +182,7 @@ man = [
     "sed '/versionadded::/d' -i {root}/man.rst",
     "rst2man.py {root}/man.rst > {root}/tmt.1",
     "rm -f {root}/man.rst",
-    ]
+]
```

Comment:
Let's keep the hanging indent consistent across the file?

Reply:
Fixed in 31f63b84